db/schema.rb

```diff
@@ -11,16 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150409130306) do
+ActiveRecord::Schema.define(version: 20150427204917) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "hstore"
```

Comment:
try and recreate the schema for the test env `bin/rake db:migrate:reset RAILS_ENV=test`